From e942a8a472cd86b4080003f0ef473d83ac477fed Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 14 Jul 2016 14:11:26 +0200
Subject: [PATCH] dts:imx6sx-score: Add config for uart2 and uart6

Add initialisation and configuration for uart2 and uart6. Although add
pinmux for these uarts.

Ticket #1291
---
 arch/arm/boot/dts/imx6sx-score.dts | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-score.dts b/arch/arm/boot/dts/imx6sx-score.dts
index 1124b43..a364af6 100644
--- a/arch/arm/boot/dts/imx6sx-score.dts
+++ b/arch/arm/boot/dts/imx6sx-score.dts
@@ -71,6 +71,18 @@
     status = "okay";
 };
 
+&uart2 {
+    pinctrl-names = "default";
+    pinctrl-0 = <&pinctrl_uart2>;
+    status = "okay";
+};
+
+&uart6 {
+    pinctrl-names = "default";
+    pinctrl-0 = <&pinctrl_uart6>;
+    status = "okay";
+};
+
 &flexcan1 {
     pinctrl-names = "default";
     pinctrl-0 = <&pinctrl_flexcan1>;
@@ -215,6 +227,21 @@
             >;
         };
 
+		pinctrl_uart2: uart2grp {
+            fsl,pins = <
+            MX6SX_PAD_GPIO1_IO06__UART2_TX		0x1b0b1
+			MX6SX_PAD_GPIO1_IO07__UART2_RX		0x1b0b1
+            >;
+        };
+
+		pinctrl_uart6: uart6grp {
+			fsl,pins = <
+			MX6SX_PAD_KEY_COL1__UART6_TX		0x1b0b1
+			MX6SX_PAD_KEY_ROW1__UART6_RX		0x1b0b1
+			>;
+		};
+
+
 		pinctrl_usdhc3: usdhc3grp {
             fsl,pins = <
             MX6SX_PAD_SD3_CMD__USDHC3_CMD		0x17069
-- 
1.9.1

